Jenkinsfile

```diff
@@ -0,0 +1,67 @@
+pipeline {
+    agent any
+
+    environment {
+        JENKINS_CREDS       = credentials('test-jenkins-user')
+        GH_CREDS            = credentials('jx-pipeline-git-github-github')
+        GHE_CREDS           = credentials('jx-pipeline-git-github-ghe')
+        GKE_SA              = credentials('gke-sa')
+
+        GIT_USERNAME        = "$GH_CREDS_USR"
+        GIT_API_TOKEN       = "$GH_CREDS_PSW"
+        GITHUB_ACCESS_TOKEN = "$GH_CREDS_PSW"
+
+        JOB_NAME            = "$JOB_NAME"
+        BRANCH_NAME         = "$BRANCH_NAME"
+        ORG                 = 'jenkinsxio'
+
+        // Build and tests configuration (run only 2 builds/tests in parallel
+        // in order to avoid OOM issue
+        PARALLEL_BUILDS = 1
+
+        // BDD tests configuration
+        GIT_PROVIDER_URL     = "https://github.beescloud.com"
+        GHE_TOKEN            = "$GHE_CREDS_PSW"
+        GINKGO_ARGS          = "-v"
+
+        JX_DISABLE_DELETE_APP  = "true"
+        JX_DISABLE_DELETE_REPO = "true"
+    }
+    options {
+        skipDefaultCheckout(true)
+    }
+    stages {
+        stage('CI Build and Test') {
+            when {
+                anyOf {
+                    environment name: 'JOB_TYPE', value: 'presubmit'
+                    environment name: 'JOB_TYPE', value: 'postsubmit'
+                    environment name: 'JOB_TYPE', value: 'batch'
+                }
+            }
+            steps {
+                dir ('/workspace') {
+                    // lets create a team for this PR and run the BDD tests
+                    sh "gcloud auth activate-service-account --key-file $GKE_SA"
+                    sh "gcloud container clusters get-credentials anthorse --zone europe-west1-b --project jenkinsx-dev"
+
+                    // image doesn't have an up-to-date jx version (it's missing the --skip-test-git-repo-clone flag)
+                    sh "wget https://github.com/jenkins-x/jx/releases/download/v1.3.875/jx-linux-amd64.tar.gz"
```

Comment:
I would use latest version. It can be retrieved from this API:

`https://api.github.com/repos/jenkins-x/jx/releases/latest`

Reply:
This is a quick workaround because the version of `jx` shipped in the image at the moment is out of date. I agree with you, but given this is a short-term fix, I'd rather not have to do JSON parsing to get the URL. This works for the moment. I was hoping that we could upgrade the build image so that it has a newer version of `jx` in it and we can just remove these lines.